<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_materias</class>
 <widget class="QWidget" name="gestion_materias">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>621</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1111</width>
    <height>621</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1111</width>
    <height>621</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f6fa;
color: #2d2d2d;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameHeader">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>81</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
		border-radius: 16px;
		background-color: transparent;
border: transparent;
}
QLabel{
	background-color: transparent;
	color: #2d2d2d;
border: transparent;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnNueva_materia">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  
	background-color: #2980b9;
    color: #FFFFFF;
    border: none;
    padding: 8px 10px;
    border-radius: 14px;
}
QPushButton:hover {
	background-color: #0D47A1
}</string>
        </property>
        <property name="text">
         <string>Nueva</string>
        </property>
        <property name="icon">
         <iconset resource="../icons/resources_ui.qrc">
          <normaloff>:/icons/nuevo_w.png</normaloff>:/icons/nuevo_w.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGestion_areas">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  
	background-color: #2980b9;
    color: #FFFFFF;
    border: none;
    padding: 5px 5px;
    border-radius: 14px;
}
QPushButton:hover {
	background-color: #0D47A1
}</string>
        </property>
        <property name="text">
         <string>Gestionar Áreas de Aprendizaje</string>
        </property>
        <property name="icon">
         <iconset resource="../icons/resources_ui.qrc">
          <normaloff>:/icons/materias_w.png</normaloff>:/icons/materias_w.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnActualizar_tabla_materias">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   background-color: white;
    color: #2980b9;
    border: 1.5px solid #2980b9;
    padding: 2px 2px;
    border-radius: 14px;
}
QPushButton:hover {
	background-color: rgb(213, 234, 255);
	border: 1.5px solid #2980b9;
}</string>
        </property>
        <property name="text">
         <string>Actualizar tabla</string>
        </property>
        <property name="icon">
         <iconset resource="../icons/resources_ui.qrc">
          <normaloff>:/icons/actualizar_b.png</normaloff>:/icons/actualizar_b.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblTitulo_materias">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>61</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>19</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #2d2d2d;
background-color: transparent;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <property name="text">
         <string>Gestión de materias</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="minimumSize">
         <size>
          <width>3</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>3</width>
          <height>61</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #2d2d2d;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblLogo_materias">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>61</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../icons/resources_ui.qrc">:/logos/logo_escuela_sinFondo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameHeader_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>51</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>51</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
		border-radius: 16px;
		background-color: transparent;
border: transparent;
}
QLabel{
	background-color: transparent;
	color: #2d2d2d;
border: transparent;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QFrame" name="frameFiltro_estu">
        <property name="minimumSize">
         <size>
          <width>147</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: white;
	border: 1.5px solid #2c3e50;
	border-radius: 12px;
}
QComboBox QAbstractItemView {
    border: 1px solid #ccc;
    border-radius: 5px;
    background-color: white;
    color: #333;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QComboBox" name="cbxFiltro_materias">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>5</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
			combobox-popup: 0;
            border-radius: 10px;
            padding: 2px 8px;
            background-color: white;
            color: #2d2d2d;
   
        }
        QComboBox:hover {
            border-color: #0D47A1;
        }
        
        QComboBox QAbstractItemView {
            background-color: white;
            border: 1.5px solid #2980b9;
            border-radius: 8px;
            color: #2d2d2d;
            outline: none;
            padding: 4px 0px;
        }
        QComboBox QAbstractItemView::item {
            min-height: 20px;
            padding: 3px 5px;
        }
        QComboBox QAbstractItemView::item:selected {
            background-color: #2980b9;
            color: white;
        }
        QComboBox QAbstractItemView QScrollBar:vertical {
            border: none;
            background: #f0f7ff;
            width: 7px;
            border-radius: 3px;
            margin: 4px 2px;
        }
        QComboBox QAbstractItemView QScrollBar::handle:vertical {
            background: #2980b9;
            border-radius: 3px;
            min-height: 20px;
        }
        QComboBox QAbstractItemView QScrollBar::add-line:vertical,
        QComboBox QAbstractItemView QScrollBar::sub-line:vertical {
            height: 0px;
        }</string>
         </property>
         <property name="iconSize">
          <size>
           <width>5</width>
           <height>5</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Todos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nombre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Abreviatura</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Estado</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Grados</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lneBuscar_materias">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>541</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #848f9d;
	color: #2d2d2d;
    border-radius: 12px;
    padding: 1px 5px;
    background-color: white;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Busqueda por cualquier dato</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblIcon_conectado_como">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>8</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #2d2d2d</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../icons/resources_ui.qrc">:/icons/usuario-activo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblConectado_como">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>161</width>
          <height>161</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>8</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #2d2d2d</string>
        </property>
        <property name="text">
         <string>Conectado como: jorged</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameTabla_materias">
     <property name="minimumSize">
      <size>
       <width>900</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>401</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frameTabla_materias {
    border: 1px solid #d5dbdb;
    border-radius: 12px;
    background-color: white;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableView" name="tableW_materias">
        <property name="styleSheet">
         <string notr="true">QTableView {
    background-color: #F7F9FC;
    gridline-color: #E3E8EF;
    color: #0B1321;
    alternate-background-color: #E3F2FD;
    selection-background-color: #2980b9;
    selection-color: #FFFFFF;
    border: 1px solid #CBD5E1;
}

/* Cabecera horizontal */
QHeaderView::section {
    background-color: #ECF0F1;   /* encabezado */
    color: #2C3E50;              /* azul oscuro */
    font-weight: bold;
    border: none;
    padding: 6px;
}

/* Cabecera vertical */
QHeaderView::section:vertical {
    background-color: #ECF0F1;
    color: #2C3E50;
    font-weight: bold;
    border: none;
    padding: 4px;
}

/* Botón de esquina (arriba a la izquierda) */
QTableCornerButton::section {
    background-color: transparent;
    border: none;
}

/* Scrollbar vertical */
QScrollBar:vertical {
    background: transparent;
    width: 10px;
    margin: 2px;
}

QScrollBar::handle:vertical {
    background: #CBD5E1;
    border-radius: 4px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background: #94A3B8;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    background: none;
    border: none;
    height: 0px;
}</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::DoubleClicked</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameSaludo_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
		border-radius: 16px;
		background-color: transparent;
border: transparent;
}
QLabel{
	background-color: transparent;
	color: #2d2d2d;
border: transparent;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="chkMostrar_inactivas_materias">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    spacing: 8px;
    color: #2d2d2d;
	background-color: transparent;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border: 2px solid #2c3e50;
    border-radius: 6px;
    background: #ffffff;
}

QCheckBox::indicator:hover {
    border: 2px solid #1565C0;
}

QCheckBox::indicator:checked {
    border: 2px solid #1565C0;
    background: #2980b9;  /* mantener blanco para que Qt pinte la palomita */
}</string>
        </property>
        <property name="text">
         <string>Mostrar inactivos</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnEditar_materia">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  
	background-color: #2980b9;
    color: #FFFFFF;
    border: none;
    padding: 8px 10px;
    border-radius: 14px;
}
QPushButton:hover {
	background-color: #0D47A1
}</string>
        </property>
        <property name="text">
         <string>Editar</string>
        </property>
        <property name="icon">
         <iconset resource="../icons/resources_ui.qrc">
          <normaloff>:/icons/edit_white.png</normaloff>:/icons/edit_white.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDesactivar_materia">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  
	
	background-color: #e74c3c;
    color: #FFFFFF;
    border: none;
    padding: 8px 6px;
    border-radius: 14px;
}
QPushButton:hover {
	background-color: #C0392B
}</string>
        </property>
        <property name="text">
         <string>Desactivar</string>
        </property>
        <property name="icon">
         <iconset resource="../icons/resources_ui.qrc">
          <normaloff>:/icons/cancelar_w2.png</normaloff>:/icons/cancelar_w2.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/resources_ui.qrc"/>
 </resources>
 <connections/>
</ui>
